pkg/services/queryhistory/database.go

```diff
@@ -265,3 +265,76 @@ func (s QueryHistoryService) unstarQuery(ctx context.Context, user *models.Signe
 
 	return dto, nil
 }
+
+func (s QueryHistoryService) migrateQueries(ctx context.Context, user *models.SignedInUser, cmd MigrateQueriesToQueryHistoryCommand) ([]QueryHistoryDTO, error) {
+	var dtos []QueryHistoryDTO
+	err := s.SQLStore.WithTransactionalDbSession(ctx, func(session *sqlstore.DBSession) error {
+		var instertError error
+
+		for _, query := range cmd.Queries {
+			// At the beginning of the loop check if there was any error and if so, break and return error
+			if instertError != nil {
+				break
+			}
+
+			if query.CreatedAt == 0 {
```

Comment:
remove this

Reply:
Removed in https://github.com/grafana/grafana/pull/47551/commits/4ea7459b93d29044e96d9c23dbfc30bf7a99e58c